pkg/southbound/gnmi/conn.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2021-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gnmi
+
+import (
+	"context"
+	"math"
+	"time"
+
+	"google.golang.org/grpc"
+
+	"github.com/onosproject/onos-lib-go/pkg/certs"
+
+	"github.com/onosproject/onos-lib-go/pkg/errors"
+
+	topoapi "github.com/onosproject/onos-api/go/onos/topo"
+	"github.com/onosproject/onos-lib-go/pkg/uri"
+
+	"crypto/tls"
+
+	"github.com/onosproject/onos-lib-go/pkg/logging"
+	baseClient "github.com/openconfig/gnmi/client"
+)
+
+var log = logging.GetLogger("southbound", "gnmi")
+
+// ConnID connection ID
+type ConnID string
+
+// Conn gNMI connection interface
+type Conn interface {
+	Client
+	ID() ConnID
+}
+
+// conn gNMI Connection
+type conn struct {
+	*client
+	id     ConnID
+	cancel context.CancelFunc
```

Comment:
I think we can remove this right? Is it even used?

Reply:
Yeah we can remove it. there is no use at the moment.

Reply:
Removed.